_data/lessons.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I'm not sure what this is trying to do that can't be done with `community_lessons.json`.

Reply:
Didn't notice it there! Removed by 333a4a9e.